### PR TITLE
cargo-mommy: init at 0.1.1

### PR DESCRIPTION
**Pull request by GoldsteinE**
###### Description of changes

Okay, first of all, sorry if it isn’t fit for merge, I’d totally understand. I’m not sure what’s the policy on small half-joke packages are, but I’m pretty sure someone will use it if it’s merged.

[cargo-mommy] is a simple `cargo` wrapper command that prints encouragements at the command line after the command itself, praising the user if the command is successful and supporting them if it’s not.

[cargo-mommy]: https://github.com/Gankra/cargo-mommy

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux (in CI)
  - [x] x86_64-darwin (in CI)
  - [x] aarch64-darwin (in CI)
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by kanashimia — APPROVED** [review]

**Comment by GoldsteinE** [comment]
Rust package contains an integration test that’d been run in CI on all platforms now.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5186,6 +5186,15 @@
     githubId = 643494;
     name = "Cillian de Róiste";
   };
+  GoldsteinE = {
```
Comment (onny) [inline maintainers/maintainer-list.nix:5189]: Please put this change into a separate commit called something like "maintainers: add GoldsteinE"

Comment (GoldsteinE) [reply maintainers/maintainer-list.nix:5189]: Done!

**Comment by onny** [comment]
Result of `nixpkgs-review pr 207398` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cargo-mommy</li>
  </ul>
</details>

**Comment by onny** [comment]
Thank you for your contribution and welcome to the maintainer list :)
